runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/KeyedTimerData.java

```diff
@@ -197,6 +197,8 @@ public void encode(KeyedTimerData<K> value, OutputStream outStream)
       }
 
       final String timerFamilyId = inStream.available() > 0 ? STRING_CODER.decode(inStream) : "";
+      final Instant outputTimestamp =
```

Comment:
Maybe add a comment saying this condition is used for version upgrade, can be cleaned up later.